getting_started.py

```diff
@@ -10,6 +13,27 @@
 """
 
 
+def pixelstring_to_numpy(string, flatten=False, integer_pixels=False):
```

Comment:
I added that here because I add a circular dependency pb when importing from dataset_tools.py.